CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## Unreleased
+## 8.0.0
 
-* BREAKING: HTML style attribute and style element, which were never supposed to be available, are forbidden.
+* BREAKING: HTML style attribute and style element, which were never supposed to be available, are forbidden. [279](https://github.com/alphagov/govspeak/pull/279)
```

Comment:
**Nitpick:** I think it's clearer when there's a `#` hash character before the PR number.

But I can see there's not a strong convention within this CHANGELOG, so you're not introducing a new problem.

```suggestion
* BREAKING: HTML style attribute and style element, which were never supposed to be available, are forbidden. [#279](https://github.com/alphagov/govspeak/pull/279)
```

Reply:
Thanks - got it.